scripts/statistics.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+const bcd = require('..');
+
+const browsers = ['chrome', 'chrome_android', 'edge', 'firefox', 'ie', 'safari', 'safari_ios', 'webview_android'];
+let stats = { total: { all: 0, true: 0, null: 0, real: 0 } };
+browsers.forEach(browser => {
+  stats[browser] = { all: 0, true: 0, null: 0, real: 0 }
+});
+
+const checkSupport = (supportData, type) => {
+  let foundType = false;
+  if (Array.isArray(supportData)) {
+    supportData.forEach(item => {
+      // if 'type' occurs once in the array, mark it as found
+      if (item.version_added === type || item.version_removed === type) {
+        foundType = true;
+      }
+    });
+  } else {
+    foundType = (supportData.version_added === type || supportData.version_removed === type);
+  }
+  return foundType;
+};
+
+const processData = (data) => {
+  if (data.support) {
+    browsers.forEach(function(browser) {
+      stats[browser].all++;
+      stats.total.all++;
+      let real_value = true;
+      if (!data.support[browser]) {
+        stats[browser].null++;
+        stats.total.null++;
+        real_value = false;
+      } else {
+        if (checkSupport(data.support[browser], null)) {
+          stats[browser].null++;
+          stats.total.null++;
+          real_value = false;
+        }
+        if (checkSupport(data.support[browser], true)) {
+          stats[browser].true++;
+          stats.total.true++;
+          real_value = false;
+        }
+      }
+      if (real_value) {
+        stats[browser].real++;
+        stats.total.real++;
+      }
+    });
+  }
+};
+
+const iterateData = (data) => {
+  for (let key in data) {
```

Comment:
Note that this will also traverse into strings and arrays, with `key` being 0, 1, 2 and so on. `processData` will get called with 1-char strings as arguments and do nothing. This is pretty harmless, just something I was bitten by when writing very similar code with different conditions in `processData`.